_sections/65-events-and-actions.md

```diff
@@ -12,9 +12,40 @@ See the W3C XForms specification [section on events](https://www.w3.org/TR/xform
 | --------------------------| ----------- |
 | <a id="event:odk-instance-first-load" href="#event:odk-instance-first-load">`odk-instance-first-load`</a><a id="event:xforms-ready"></a>            | dispatched the first time an instance is loaded |
 | <a id="event:xforms-value-changed" href="#event:xforms-value-changed">`xforms-value-changed`</a>    | dispatched after an instance data node's value changes. |
+| <a id="event:odk-new-repeat" href="#event:odk-new-repeat">`odk-new-repeat`</a>	| dispatched when a new instance of a repeat is added to the primary instance and before recomputation of `calculates`, etc. Actions triggered by `odk-new-repeat` must be nested in the repeat form control. <a href="#the-odk-new-repeat-event">See more</a>.
 
 *Note: `xforms-ready` was previously documented as the event dispatched the first time an instance is loaded. Since that definition does not match the W3C XForms event with the same name, it was deprecated in favor of `odk-instance-first-load`.*
 
+#### The `odk-new-repeat` event
+The `odk-new-repeat` event is dispatched when a new instance of a repeat is added to the primary instance and before recomputation of `calculates`, `constraints`, etc. Actions triggered by `odk-new-repeat` must be nested in the repeat form control.
+
```

Comment:
The paragraph below is indeed very good and useful to record for posterity, but I don't think it should be in the spec. Maybe we can at least paste it into the GitHub issue?

Reply:
👍